catalog/core/catalog-core-subscriptionstore/subscriptionstore-impl/pom.xml

```diff
@@ -42,6 +42,26 @@
             <artifactId>cache-api</artifactId>
             <version>1.0.0</version>
         </dependency>
+        <dependency>
+            <groupId>org.cache2k</groupId>
+            <artifactId>cache2k-all</artifactId>
+            <version>1.0.0.CR4</version>
+        </dependency>
+        <dependency>
+            <groupId>org.cache2k</groupId>
+            <artifactId>cache2k-jcache</artifactId>
+            <version>1.0.0.CR4</version>
+        </dependency>
+        <dependency>
+            <groupId>org.cache2k</groupId>
+            <artifactId>cache2k-api</artifactId>
+            <version>1.0.0.CR4</version>
+        </dependency>
+        <dependency>
+            <groupId>org.cache2k</groupId>
+            <artifactId>cache2k-jcache-api</artifactId>
+            <version>1.0.0.CR4</version>
+        </dependency>
```

Comment:
If they are using a tool to search for it first, which they should, then it shouldn't matter where it is. I'm concerned if it is discovered in a top parent POM someone will interpret that as a green light to use it everywhere.